deploy/charts/trust-manager/values.yaml

```diff
@@ -68,6 +68,9 @@ app:
       type: ClusterIP
 
     tls:
+      # -- Whether to issue an "isolated" cert, which removes the requirement for cert-manager to be installed by using Helm to issue a webhook certificate.
```

Comment:
It would be great to explain why this approach is fine as long as you upgrade trust-manager at least every once every 10 years. Am I understanding correctly?

Reply:
Great, the new comment answers all the of questions I had.